constitution.tex

```diff
@@ -770,6 +770,20 @@
 \item Candidates must be Active Members
 \end{enumerate}
 
+\asubsection{Waiving of Requirements}
+\begin{enumerate}
+  \item The following requirements may be waived as decided by a Two-Thirds Immediate Vote as described in \ref{Two-Thirds} and \ref{Immediate Vote}:
+    \begin{enumerate}
+      \item Candidates must have at least one full semester of House membership as an Active Member
+      \item Candidates must reside on floor during the term of office
```

Comment:
nit: Swap this order since it follows the opposite order in "Qualifications to be the Chairperson, Evaluations Director"